.github/workflows/ci_tests_dev.yaml

```
@@ -106,6 +106,7 @@ jobs:
 
       # Pull baseline image data from dvc remote (DAGsHub)
       - name: Pull baseline image data from dvc remote
+        if: github.event_name == 'schedule'
         run: |
           ORIGINAL_BRANCH=$(git branch --show-current)
           # Pull down GMT 6.4 baseline images from the gmt-6.4 branch
```

Should the `gmt-6.4` branch have a Pull Request associated with it? Or do we just keep pushing to that `gmt-6.4` branch directly?

---

> Should the `gmt-6.4` branch have a Pull Request associated with it?

Opening a PR for gmt-6.4 sounds a good idea.

---

Ok, then we can reference the `gmt-6.4` with a GitHub URL, and add it to the file here:

```suggestion
          # Pull down GMT 6.4 baseline images from the gmt-6.4 branch
          # https://github.com/GenericMappingTools/pygmt/pull/<number>
```

---

I can't open a PR for the gmt-6.4 branch because currently it's the same as the main branch. So we can't know the exact PR number now. Maybe we can assume that the next PR (#1883) comes from the gmt-6.4 branch.

---

Could have `gmt-6.4.` target a different branch first (e.g. `update-baseline-images-for-gmt-dev`) and then switch it to target the `main` branch later?

---

Great idea. The PR is now at https://github.com/GenericMappingTools/pygmt/pull/1883.

---

Not sure why the above suggestion doesn't work, let's try again.
```suggestion
          # Pull down GMT 6.4 baseline images from the gmt-6.4 branch
          # https://github.com/GenericMappingTools/pygmt/pull/1883
```